addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -279,6 +271,20 @@ function factory(dependencies) {
             this._toggleVideoBroadcast({ type: 'user-video' });
         }
 
+        async unDeafen() {
+            await this._setDeafState(false);
+        }
+
+        async unMute() {
+            if (this.audioTrack) {
+                this.currentRtcSession.updateAndBroadcast({ isMuted: false });
+            } else {
+                // if we don't have an audioTrack, we try to request it again
+                await this.updateLocalAudioTrack(true);
+            }
+            await this.async(() => this._updateLocalAudioTrackEnabledState());
```

Comment:
this.async is effectively useless (it will crash anyway, just with a different error), we should stop using that method